### DISCUSSION TITLE
404: NOT_FOUND

### DISCUSSION THREAD

**Discussion by zubisss9** [vercel/community#3175] [Help] [answered]
### Summary

404: NOT_FOUND
Code: DEPLOYMENT_NOT_FOUND
ID: bom1::smkv5-1688979187657-9e0c533fc985
![404](https://github.com/vercel/community/assets/133433832/95f98064-c9b4-41ec-b119-7259e3ede86e)

please solve this issues. This is the URL of my website. https://zoofeed.live/

### Example

This is the URL of my website. https://zoofeed.live/

### Steps to Reproduce

please solve this issues. This is the URL of my website. https://zoofeed.live/

**Comment by swarnava** [comment, marked as answer]
Hi,

I don't see domain zoofeed.live is attached to any projects. Are you using this instruction https://vercel.com/docs/concepts/projects/domains/add-a-domain ?

**Comment by zubisss9** [comment]
No I am not using

**Reply by swarnava** [reply]
You need to follow above instruction in order to setup custom domain with Vercel.
